guides/moving-to-oss.md

```diff
@@ -0,0 +1,235 @@
+| Kubernetes (built-in database)          | ✅                                                                                                                   | ❌                                                                                                                                                                    |
+| Docker deployment                       | ✅                                                                                                                   | ✅                                                                                                                                                                    |
+| VM deployment                           | ❌                                                                                                                   | ✅ [system packages](https://coder.com/docs/coder-oss/latest/install#system-packages) and [prebuilt binaries](https://coder.com/docs/coder-oss/latest/install#manual) |
+| Built-in PostgreSQL                     | ✅                                                                                                                   | ✅                                                                                                                                                                    |
+| Built-in TLS tunnel                     | ❌                                                                                                                   | ✅                                                                                                                                                                    |
+| External PostgreSQL support             | ✅                                                                                                                   | ✅ ([configuration flag](https://coder.com/docs/coder-oss/latest/install/configure))                                                                                  |
+| External TLS documentation              | ✅ (via [cert-manager](https://coder.com/docs/coder/latest/guides/tls-certificates))                                 | ⌛ [#3518](https://github.com/coder/coder/issues/3518)                                                                                                                |
+| **Multi region/cloud (workspaces)**     | ✅ [Workspace providers](https://coder.com/docs/coder/latest/admin/workspace-providers) support additional clusters. | ✅ [Templates](https://coder.com/docs/coder/latest/admin/templates) can provision resources in any clouds, clusters, or region                                        |
+| **Multi region/cloud (provisioning)**   | ❌                                                                                                                   | ⌛ [#44](https://github.com/coder/coder/issues/44)                                                                                                                    |
```

Comment:
I don't understand this feature title in relation to the linked issue.

Reply:
I removed it. I don't think it necessarily makes any sense here.